Subset_Sum/Subset_Sum.py

```diff
@@ -0,0 +1,74 @@
+''' Subset Sum Problem: Given an array of elements and a sum,
+is it possible to form the given sum using some or all of the
+elements of array i.e using any subset of given array. 
+Example: 1, 7, 3, 5, 6 and sum = 14, it is possible to obtain 
+the sum 14 by taking the elements 1, 7 and 6 '''
+
+# Function to compute whether a subset of array exists 
+# having the given sum
+def subsetSum(arr, size, sum) :
+
+    # A 2D Matrix containing boolean values, true and false
+    dp = []
+    for i in range(0, size+1):
```

Comment:
Add spaces around all operators in this file. Other files are fine.

Reply:
Done